Dockerfile

```diff
@@ -1 +1,7 @@
 FROM hone/mruby-cli
+
+RUN groupadd -r mruby -g 1000 && \
+    useradd -d /home/mruby/ -u 1000 -g mruby -r -s /sbin/nologin -c "Docker image mruby user" mruby && \
+    chown -R mruby:mruby /home/mruby/
```

Comment:
This should go in the upstream container?

I thought we had issues with this on OS X because how boot2docker deals with permissions